examples/minimal-logs/tamrvm.tf

```diff
@@ -20,10 +20,11 @@ module "tamr-vm" {
   aws_role_name               = format("%s-tamr-ec2-role", var.name-prefix)
   aws_instance_profile_name   = format("%s-tamr-ec2-instance-profile", var.name-prefix)
   aws_emr_creator_policy_name = format("%sEmrCreatorPolicy", var.name-prefix)
-  s3_policy_arns = [
+  additional_policy_arns = [
     module.s3-bucket.rw_policy_arn,
     "arn:aws:iam::aws:policy/CloudWatchAgentServerPolicy"
   ]
+  s3_policy_arns    = []
```

Comment:
I'd leave this out. A new example doesn't need to include deprecated variables (even though it's empty, it brings confusion)

Reply:
Good catch. This was not supposed to be there. It is going to be in variables with the DEPRECATED flag and default to empty.